app/javascript/packs/task_form.jsx

```diff
@@ -1,22 +1,96 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import PropTypes from "prop-types";
+import axios from "axios";
 
-import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Form, FormGroup, Label, Input,
-  FormText,ButtonDropdown, DropdownToggle, DropdownMenu, DropdownItem  } from "reactstrap";
 
-class ModalExample extends React.Component {
+import { Button, Modal, ModalHeader, ModalBody, Form, FormGroup, Label, Input
+  ,ButtonDropdown, DropdownToggle, DropdownMenu, DropdownItem  } from "reactstrap";
+
+
+
+
+class Task_form extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       modal: false,
-      dropdownOpen: false
+      dropdownOpen: false,
+      data: {
+        title: "",
+        desc: "",
+        project_id: 1,
```

Comment:
use `null` as initial value instead of `1` because it can be misleading

Reply:
I set it at 1 becouse on single page we create task for current project in future this value will be set according to project index